src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = 0;
```

Comment:
instead of variable redeclaring you can just set a default result and return this value for conditions (look an example below)